Library/Formula/phantomjs.rb

```diff
@@ -26,11 +14,14 @@ class Phantomjs < Formula
     sha256 "45091efed57f4de5f04810a874e050206ae587ac85e17892f570e0e7eb50b977" => :mountain_lion
   end
 
+  depends_on "openssl"
+
   def install
-    system "./build.sh", "--confirm", "--jobs", ENV.make_jobs,
-      "--qt-config", "-openssl-linked"
+    inreplace "src/qt/preconfig.sh", "QT_CFG+=' -openssl -openssl-linked'",
+              "QT_CFG+=' -openssl -openssl-linked -I #{Formula["openssl"].opt_include} -L #{Formula["openssl"].opt_lib}'"
+    system "./build.sh", "--confirm", "--jobs", ENV.make_jobs
```

Comment:
We don't have `build.sh` anymore. We switched to the unified Python script to build PhantomJS.